src/scripts/main.js

```diff
@@ -22,7 +22,22 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  if (!element || !data) {
```

Comment:
```suggestion
  if (!element || !data || !Object.keys(data).length) {
```
do not create ul if data is empty object